skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SystemTheme.kt

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.skiko
+
+enum class SystemTheme {
+    DARK,
+    LIGHT,
+    UNKNOWN
+}
+
+val currentSystemTheme: SystemTheme
```

Comment:
Can we write something like `addCurrentSystemThemeListener"?

Reply:
As discussed, let's use generic configuration callback.